src/components/accommodation/AccommodationFormFields.js

```diff
@@ -5,43 +5,93 @@ import { FormikApiErrors } from "components/atoms/form/FormikApiErrors";
 import Accommodation from "models/Accommodation";
 import { getFormattedDate } from "shared/datetime";
 
+const objectAssignMapped = (object, source, map) => {
+    for (const [objectKey, sourceKey] of Object.entries(map)) {
+        object[objectKey] = source[sourceKey];
+    }
+    return object;
+};
+
 class AccommodationFormFields {
+    static UUID = "uuid";
+    static ID = "id";
+    static STATUS = "status";
     static ADDRESS_VOIVODESHIP = "addressVoivodeship";
     static ADDRESS_CITY = "addressCity";
     static ADDRESS_LINE = "addressLine";
     static ADDRESS_ZIP = "addressZip";
-    // static CREATED_AT = 'createdAt';
     static COMMENTS = "comments";
-    static DESCRIPTION = "description";
-    static ID = "id";
-    static IS_VERIFIED = "isVerified";
-    static HOST_EMAIL = "hostEmail";
-    static HOST_NAME = "hostName";
-    static HOST_PHONE = "hostPhone";
     static PETS_PRESENT = "petsPresent";
     static PETS_ALLOWED = "petsAllowed";
-    static STATUS = "status";
     static VACANCIES_TOTAL = "vacanciesTotal";
-    static VACANCIES_TAKEN = "vacanciesTaken";
-    static UUID = "uuid";
-    static VOLUNTEER_NAME = "volunteerName";
+    static VACANCIES_FREE = "vacanciesFree";
```

Comment:
I'm aware we have vacanciesFree on the API side, but I've spoken to Korvin from Salam Lab, and from the volunteers' and UX perspective "Taken places" make more sense. Please revert this change back and do the proper translation when converting Accommodation to API model. Ideally this could be solved within a model itself by capabilities offered by `ta-json`.